### PR TITLE
Include possibility to use R

### PR DESCRIPTION
**Pull request by jpy-luke**
Introduced the usage of R with Lauri Mehtätalo's lmfor_volume function for forest stand volume calculation.

This work server as first example of how to use R with Python, and bind it to a simulator operation.
Also added GitHub Action additions to allow running related tests on the test run pipeline.

### CODE REVIEW

**Comment by taplem** [comment]
Do we need the R tests? It seems to me that they're only testing if rpy2 works, and rpy2 already has a quite comprehensive test suite.

File: forestry/r_utils.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    source_data = {
+        'height': robjects.FloatVector([tree.height for tree in stand.reference_trees]),
+        'breast_height_diameter': robjects.FloatVector([tree.breast_height_diameter for tree in stand.reference_trees]),
+        'degree_days': robjects.FloatVector([stand.degree_days for _ in range(len(stand.reference_trees))]),
+        'species': robjects.StrVector([lmfor_species_map[tree.species] for tree in stand.reference_trees]),
+        'model_type': robjects.StrVector(['scanned' for _ in range(len(stand.reference_trees))])
+    }
+    df = robjects.DataFrame(source_data)
+    volumes = list(robjects.r['compute_tree_volumes'](df))
+    total_volume = round(sum(volumes), 1)
```
[the remark this answers is not in the document]
Comment (jpy-luke) [reply forestry/r_utils.py:37]: You are totally right. I'll address this.

**Comment by jpy-luke** [comment]
About the R tests, it is ok practice to investigate how to use a library via tests. They serve as examples for the next guy, and also help detect potential breakages and API changes in library version updates.

**Review by taplem — APPROVED** [review]

File: tests/app/r_test.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import unittest
+import rpy2
+import rpy2.robjects as robjects
+from rpy2.robjects.packages import importr
+import rpy2.robjects.packages as rpackages
+
+
+class RTest(unittest.TestCase):
+    def test_init(self):
+        print(rpy2.__version__)
```
Comment (dmela) [inline tests/app/r_test.py:10]: What is the point if test has no assertion?

**Pull request merged**